app/models/manageiq/providers/cisco_intersight/physical_infra_manager/vm.rb

```diff
@@ -1,5 +1,5 @@
 module ManageIQ::Providers::CiscoIntersight
-  class PhysicalInfraManager::Vm < ManageIQ::Providers::PhysicalInfraManager::Vm
+  class PhysicalInfraManager::Vm < ManageIQ::Providers::InfraManager::Vm
```

Comment:
`PhysicalInfra` is different from `InfraManager` I'm not sure we want to simply subclass `InfraManager::Vm`.  Ideally we would create a `ManageIQ::Providers::PhysicalInfraManager::Vm < ::Vm` subclass in core and subclass that here.

Reply:
Makes sense to me.  Should I drop this part for now?  I can keep this local until it's fixed while I do more zeitwerk work.  Let me know.

Reply:
Yes I'll throw a core PR up quickly so this will be a valid class

Reply:
https://github.com/ManageIQ/manageiq/pull/22473

Reply:
testing it now...

Reply:
works now... the constant is defined 👍

Reply:
I've removed this change.